src/query.ts

```diff
@@ -138,6 +138,31 @@ export class Query<T> {
     );
   }
 
+  /**
+   * Add sql statements to the batch only
+   *
+   * @returns The parameters for the batch
+   */
+  public addToBatch(parameters: BindingValue[]): BindingValue[] {
+    this.parameters = this.parameters ?? [];
+    if (!Array.isArray(parameters)){
+      throw new Error("Parameter 'parameters' must be a 2d array of parameters to the query");
+    } else {
+      for (const parameter of parameters){
+        if (!Array.isArray(parameter)){
+          throw new Error("Parameter 'parameters' must be a 2d array of parameters to the query");
+        }
+      }
+    }
+
+    for (const parameter of parameters){

```

Comment:
Replace with `this.parameters.push(...parameters)`

Reply:
Suggestion added